src/cli/args.rs

```diff
@@ -97,3 +97,158 @@ pub enum Subcommand {
         tree: bool,
     },
 }
+
+#[cfg(test)]
+impl Default for CliArgs {
```

Comment:
I think we can just put this in a function inside `mod tests`

Reply:
:+1: changed and force-pushed.